lib/util/lifted/influx/httpd/handler_compress.go

```diff
@@ -0,0 +1,237 @@
+package httpd
```

Comment:
The Apache license header is missing. 
Refer：https://github.com/openGemini/openGemini/blob/main/app/ts-monitor/run/handler_metrics.go

Reply:
Resloved